cmd/ocp.go

```diff
@@ -80,17 +80,22 @@ func openShiftCmd() *cobra.Command {
 		wh = workloads.NewWorkloadHelper(workloadConfig, ocpConfig, kubeClientProvider)
 		wh.MetricsMetadata = make(map[string]interface{})
 		envVars := map[string]string{
-			"UUID":           workloadConfig.UUID,
-			"ES_SERVER":      esServer,
-			"ES_INDEX":       esIndex,
-			"LOCAL_INDEXING": fmt.Sprintf("%v", *localIndexing),
-			"QPS":            fmt.Sprintf("%d", QPS),
-			"BURST":          fmt.Sprintf("%d", burst),
-			"GC":             fmt.Sprintf("%v", gc),
-			"GC_METRICS":     fmt.Sprintf("%v", gcMetrics),
+			"UUID":       workloadConfig.UUID,
+			"QPS":        fmt.Sprintf("%d", QPS),
+			"BURST":      fmt.Sprintf("%d", burst),
+			"GC":         fmt.Sprintf("%v", gc),
+			"GC_METRICS": fmt.Sprintf("%v", gcMetrics),
 		}
+		envVars["LOCAL_INDEXING"] = fmt.Sprintf("%v", *localIndexing)
 		if alerting {
 			envVars["ALERTS"] = "alerts.yml"
+		} else {
+			envVars["ALERTS"] = ""
+		}
+		// If metricsEndpoint is not set, use values from flags
```

Comment:
Are we making sure that we are giving higher precedence to `metricEndpoint` over ES CLI args in the kube-burner core repo too?

Reply:
There're not ES CLI args in kube-burner core, they only exist in `kube-burner-ocp` (for now)

Reply:
Looks like there is one: https://github.com/kube-burner/kube-burner/blob/main/cmd/kube-burner/kube-burner.go#L356. Which can be removed I guess. wdyt?

Reply:
Yes it doesn't. But when we have metric endpoint do we want to keep es-server as a CLI arg?

Reply:
Not sure if Im following you... as I said before, the ES_SERVER and ES_INDEX env vars are not used by kube-burner core. I've added a small hint in the `--metrics-endpoint` help to note this behavior